packages/compass-aggregations/src/components/pipeline-toolbar-leafygreen/pipeline-actions.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import type { ConnectedProps } from 'react-redux';
+import type { Dispatch } from 'redux';
+import { Button, css, spacing } from '@mongodb-js/compass-components';
+
+import type { RootState } from '../../modules';
+import { exportToLanguage } from '../../modules/export-to-language';
+import { setIsModified } from '../../modules/is-modified';
+import { savingPipelineOpen } from '../../modules/saving-pipeline';
+import { saveCurrentPipeline } from '../../modules/saved-pipeline';
+const pipelineActionsContainerStyles = css({});
+
+const buttonStyles = css({
+  marginLeft: spacing[1],
+  marginRight: spacing[1],
+});
+
+const PipelineActions: React.FunctionComponent<PipelineActionsProps> = ({
+  name,
+  onExportToLanguage,
+  onSavePipeline,
+}) => {
+  return (
+    <div className={pipelineActionsContainerStyles}>
+      <Button className={buttonStyles} variant="primary">
+        Run
+      </Button>
+      <Button className={buttonStyles} onClick={() => onSavePipeline(name)}>
```

Comment:
What do you think about splitting this PR in 2? One that only enables the FF and renders an empty component and more following PR that would add features and interactions here?

If we are adding features here I'd expect to see also related tests for it in the same PR, in this case, the number of changes may become significant, if possible I'd consider breaking it in multiple PRs that are going to merge value completed with tests.